<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test2</class>
 <widget class="QWidget" name="Test2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QPushButton" name="but_open">
     <property name="text">
      <string>打开Excel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="but_close">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="com_sheets">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="but_querySheet">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>查询工作表名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="line_sheetName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>sheet类型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>sheet位置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>sheet名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="com_sheetType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>ST_WorkSheet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ST_ChartSheet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ST_DialogSheet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ST_MacroSheet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spin_index">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="but_addSheet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>创建工作表</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="but_insert">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>插入工作表</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="but_select">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>选择工作表</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="but_move">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>移动工作表</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="but_delete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除工作表</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>新Sheet名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="line_newSheetName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="but_rename">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>重命名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="but_copy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>复制工作表</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>154</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
